devops-guides/how-to-setup-a-node-with-docker.md

```diff
@@ -61,6 +61,8 @@ docker-compose up -d
 
 This will run two separate containers. One for Core itself and another one for PostgreSQL.
 
+Remember that the API will be available only after the relay finishes the update. Until there, the container will respond with `Connection reset by peer` to any request made any of its ports. 
```

Comment:
```suggestion
The public API won't be available until a relay is fully synchronized with the network and blockchain. All requests before this will result in a connection reset because the port is not yet bound to an application.
```